.gitmodules

```diff
@@ -22,3 +22,7 @@
 [submodule "3rdparty/googletest"]
 	path = 3rdparty/googletest
 	url = https://github.com/google/googletest.git
+[submodule "3rdparty/mkldnn"]
+	path = 3rdparty/mkldnn
+	url = https://github.com/ashokei/mkl-dnn.git
```

Comment:
This seems like a private fork.

Reply:
that's right. It's mainly used for testing. The main repo of MKLDNN uses "curl" to download MKLML library and the Jenkins docker doesn't contain curl. I can't build a new docker image, as you know.

Reply:
Feel free to add curl to the docker container. I don't think we should have private forks as third party dependency.